src/views/HomeView.vue

```diff
@@ -42,18 +72,114 @@
 </template>
 
 <script>
-import { mapGetters } from 'vuex';
+import _ from "lodash";
+import { mapGetters } from "vuex";
+
 export default {
   computed: {
-    ...mapGetters(['appData'])
+    ...mapGetters({
+      appData: "appData",
+      latestPosts: "blog/getLatestPosts"
+    }),
+    carouselSlides() {
+      return this.latestPosts.slice(0, 3);
+    },
+    otherLatestPosts() {
+      return this.latestPosts.slice(3, 6);
+    },
+    currentSlidePost() {
```

Comment:
since the carousel only indexes the current slide, we use this to reference the actual post object for displaying what is currently being featured in the form of title and subTitle underneath the carousel